modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -14020,6 +14027,8 @@ select-user-tools=Select User Tools
 select-users-to-associate-with-the-action-x=Select users to associate with the action {0}.
 select-users-to-associate-with-the-following-actions-x=Select users to associate with the following actions: {0}
 select-version=Select Version
+select-vocabularies=Select Vocabularies
+select-vocabularies-configuration-description=Select vocabularies if you want to control which categories to be displayed from the results. By default, all categories from any vocabularies will be shown.
```

Comment:
:+1: LGTM. but @rbohl may have a better version in his mind.

Reply:
I think what I have below is a bit clearer (but it's possible I've messed up the details--please review carefully for accuracy)
> Select the vocabularies containing the categories to display. By default, all categories from every vocabulary are considered.

I went with "considered" instead of "shown" because, if I understand right, they won't necessarily be shown, will they? If there's no results matching a certain category, it won't be shown in the facet. If I understand wrong let me know. I'm having a hard time dropping into context.

Reply:
>I went with "considered" instead of "shown" because, if I understand right, they won't necessarily be shown, will they? If there's no results matching a certain category, it won't be shown in the facet. I

Your understanding is correct, @rbohl. In the docs, we can say about this even more, like "considered means, a category will be shown if there are result matching for the keyword (and the other selected facet terms) and those results have that category.

Reply:
I like Russ' version.

Reply:
ya this one is a little tricky to word... another variation could be:

The categories returned by a search will only be displayed by this widget if they are within the vocabularies selected here.

Reply:
>By default, all categories from every vocabulary are considered.

not sure if we need this "by default" text... if permissions are involved, might not be "all categories" from "every vocabulary"?

Reply:
Yeah, removing that "By default ..." sentence should be fine.

Reply:
To be clear, it's now this?

> Select the vocabularies containing the categories to display. All categories from every vocabulary are considered.

Reply:
I think we'd remove the second sentence, not just "by default".

> Select the vocabularies containing the categories to display.

Reply:
@rbohl oh ok the whole sentence... got it, thanks!!